time="2021-07-27 15:36:40.040" level=error msg="exit status 12021/07/27 15:36:40 Failed to create file system for \"Google_Drive:rclone-gg-zzq\": couldn't find root directory ID: googleapi: Error 401: Invalid Credentials, authError\n" func=migrates3/service.RcloneGs2SyncS3Drive file="/home/zzq/go/src/gs2migrations3/service/rcloneService.go:15"
time="2021-07-27 15:36:40.041" level=error msg="exit status 1" func=migrates3/service.TimeCompare file="/home/zzq/go/src/gs2migrations3/service/rcloneService.go:36"
time="2021-07-27 15:36:40.041" level=error msg="exit status 1" func=migrates3/cloud/google.SyncDataWithGoogleDrive file="/home/zzq/go/src/gs2migrations3/cloud/google/googledrive.go:26"
time="2021-07-27 16:43:57.643" level=error msg="exit status 12021/07/27 16:43:57 Failed to create file system for \"Google_Drive:rclone-gg-zzq\": couldn't find root directory ID: googleapi: Error 401: Invalid Credentials, authError\n" func=migrates3/service.GetGoogleTime file="/home/zzq/go/src/gs2migrations3/service/rcloneService.go:104"
time="2021-07-27 16:44:25.608" level=error msg="exit status 1" func=migrates3/cloud/google.SyncDataWithGoogleDrive file="/home/zzq/go/src/gs2migrations3/cloud/google/googledrive.go:12"
time="2021-07-27 16:47:02.254" level=error msg="exit status 12021/07/27 16:46:01 Failed to create file system for \"Google_Drive:rclone-gg-zzq\": couldn't find root directory ID: googleapi: Error 401: Invalid Credentials, authError\n" func=migrates3/service.GetGoogleTime file="/home/zzq/go/src/gs2migrations3/service/rcloneService.go:104"
time="2021-07-27 16:47:02.255" level=error msg="exit status 1" func=migrates3/cloud/google.SyncDataWithGoogleDrive file="/home/zzq/go/src/gs2migrations3/cloud/google/googledrive.go:12"
